.gitignore

```diff
@@ -1,4 +1,5 @@
 # ThePlatform
+Secure.py
```

Comment:
remove.

There is a directory names secure/. Put a file in there named cookies.py. put a dict in there named coinbase.